src/bubbleSort.js

```diff
@@ -0,0 +1,17 @@
+//time complexity: O(n^2)
+
+export default function bubbleSort(arr){
+  var swapped
+  do {
+    swapped = false
+    for(let i=0; i < arr.length; i++){
+      if(arr[i] > arr[i+1]){
+        let temp = arr[i]
+        arr[i] = arr[i+1]
+        arr[i+1] = temp
+        swapped = true
+      }
+    }
+  } while(swapped)
+  return arr
+}
```

Comment:
This is pretty cool using the swapped boolean. I noticed my chai library forced me to test for conditions. You can take a look if you wanted to see what sort of tests you might run at the beginning to knock those out of the way early on: https://github.com/mantinone/core-algorithms/blob/master/src/bubbleSort.js